README.md

```diff
@@ -21,15 +22,15 @@ sudo ./cgroupv2_setup.sh
 <b>Run the serverless application :</b>
 
 ```
-cargo build --release --target x86_64-unknown-linux-musl && sudo ./target/x86_64-unknown-linux-musl/release/serverlessrust
+cargo build --release --target x86_64-unknown-linux-musl && sudo ./target/x86_64-unknown-linux-musl/release/serverlessrust --signer ./path/to/signer
```

Comment:
Maybe add the sample signer file.

Reply:
We could mention that you can generate signer from the https://github.com/marlinprotocol/keygen repo

Reply:
I meant, you can mention  that secp key is required or something.

Also, update the  `serverlessrust` to `oyster-serverless`. Check .toml file.

And similarly `Dockerfile` & `supervisord.conf` should be updated.

Reply:
Done